pulsar-client-cpp/lib/auth/AuthOauth2.cc

```diff
@@ -300,7 +364,7 @@ AuthenticationPtr AuthOauth2::create(const std::string& authParamsString) {
 
 AuthenticationPtr AuthOauth2::create(ParamMap& params) { return AuthenticationPtr(new AuthOauth2(params)); }
 
-const std::string AuthOauth2::getAuthMethodName() const { return "token"; }
+const std::string AuthOauth2::getAuthMethodName() const { return "oauth2"; }
```

Comment:
@zymap Could we keep it as token? client get token finally from oauth2 server, and broker side could use token to talk with client

Reply:
Sure.